<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="newSaveAction"/>
    <addaction name="aboutAction"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="cutAction"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="selectAction"/>
    <addaction name="fillAction"/>
    <addaction name="fontAction"/>
    <addaction name="eraserAction"/>
    <addaction name="paintBrushAction"/>
    <addaction name="shapeAction"/>
    <addaction name="colorAction"/>
    <addaction name="rotateAction"/>
    <addaction name="resizeAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="pasteAction"/>
   <addaction name="copyAction"/>
   <addaction name="cutAction"/>
   <addaction name="eraserAction"/>
   <addaction name="fontAction"/>
   <addaction name="fillAction"/>
   <addaction name="paintBrushAction"/>
   <addaction name="colorAction"/>
   <addaction name="shapeAction"/>
   <addaction name="rotateAction"/>
   <addaction name="resizeAction"/>
   <addaction name="separator"/>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/new.png</normaloff>:/new/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/save.png</normaloff>:/new/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="newSaveAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/save as.png</normaloff>:/new/images/save as.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/about.png</normaloff>:/new/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/quit.png</normaloff>:/new/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/open.png</normaloff>:/new/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/cut.png</normaloff>:/new/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/copy.png</normaloff>:/new/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/paste.png</normaloff>:/new/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="selectAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/cut out.png</normaloff>:/new/images/cut out.png</iconset>
   </property>
   <property name="text">
    <string>选择裁剪</string>
   </property>
  </action>
  <action name="fillAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/fill.png</normaloff>:/new/images/fill.png</iconset>
   </property>
   <property name="text">
    <string>填充</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/font.png</normaloff>:/new/images/font.png</iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="eraserAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/rubber.png</normaloff>:/new/images/rubber.png</iconset>
   </property>
   <property name="text">
    <string>橡皮擦</string>
   </property>
  </action>
  <action name="paintBrushAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/brush.png</normaloff>:/new/images/brush.png</iconset>
   </property>
   <property name="text">
    <string>画刷</string>
   </property>
  </action>
  <action name="shapeAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/shape.png</normaloff>:/new/images/shape.png</iconset>
   </property>
   <property name="text">
    <string>形状</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/color.png</normaloff>:/new/images/color.png</iconset>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="rotateAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/rotate.png</normaloff>:/new/images/rotate.png</iconset>
   </property>
   <property name="text">
    <string>旋转</string>
   </property>
  </action>
  <action name="resizeAction">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/images/resize.png</normaloff>:/new/images/resize.png</iconset>
   </property>
   <property name="text">
    <string>重新调整大小</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
